lib/db/leveldb_dbinstance.go

```diff
@@ -465,7 +463,7 @@ func (db *Instance) dropFolder(folder []byte) {
 	dbi.Release()
 }
 
-func (db *Instance) dropDeviceFolder(device, folder []byte, globalSize *sizeTracker) {
+func (db *Instance) dropDeviceFolder(device, folder []byte, globalSize *metadataTracker) {
```

Comment:
I would rename `globalSize` to `meta` as well here, otherwise it may mislead someone to think it is separate from the "general" metadata tracker.